### DISCUSSION TITLE
Does RiMusic fully work in airplane mode (no internet) ?

### DISCUSSION THREAD

**Discussion by rramirezg22** [fast4x/RiMusic#3524] [Q&A]
Does RiMusic fully work in airplane mode (no internet) ?

**Comment by fast4x** [comment]
It work with alla cached or downloaded or on device songs

**Comment by rramirezg22** [comment]
most of the time I get an error message "No network connection"

**Reply by fast4x** [reply]
send video please

**Comment by rramirezg22** [comment]
video:

https://github.com/user-attachments/assets/b24ef660-43da-40cb-9fab-c1a95c0e9de8

**Comment by fast4x** [comment]
You can play local or cached or. downloaded songs, other are online and require internet....

**Comment by fast4x** [comment]
Download all songs that you want listen offline and enjoy without data connection.
